### PR TITLE
Updated root CMakeLists.txt to include 2 flags for enabling arm64 windows build

### PR DESCRIPTION
**Pull request by moyo1997**

### CODE REVIEW

**Comment by mrkubax10** [comment]
Could you fix build issues?

**Comment by moyo1997** [comment]
> Could you fix build issues?

Yes, fixed them.

File: CMakeLists.txt [hunk 1]
```diff
@@ -50,6 +50,13 @@ set(CMAKE_CXX_STANDARD 14)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_CXX_EXTENSIONS OFF)
 
+# This should be set only with windows Visual Studio builds
+string(TOLOWER "${CMAKE_GENERATOR_PLATFORM}" PLATFORM)
+if(${PLATFORM} MATCHES "arm64")
```
[the remark this answers is not in the document]
Comment (moyo1997) [reply CMakeLists.txt:55]: sorry, I mean with visual studio generator not visual studio IDE. 
From the docs, it look like supertux is to be built with visual studio generators right?

[the remark this answers is not in the document]
Comment (mrkubax10) [reply CMakeLists.txt:55]: Yes but only for Windows. What if someone builds for arm64 but on Linux for example?

**Pull request closed without merging**